webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableSelectors.js

```diff
@@ -64,6 +65,10 @@ export const selectSelectedIds = state =>
 export const selectIsAllSelected = state =>
   selectInsightsCloudTable(state).isAllSelected || false;
 
+export const selectIsAllSelectedQuery = state =>
+  selectQuery(state).isSelectAll === true ||
+  selectQuery(state).isSelectAll === 'true';
```

Comment:
the `query` values are always strings,

```suggestion
export const selectIsAllSelectedQuery = state => selectQuery(state).isSelectAll === 'true';
```